admin_user_integration_test.go

```diff
@@ -37,6 +38,9 @@ func TestAdminUsers_List(t *testing.T) {
 				PageSize:   100,
 			},
 		})
+
+		fmt.Printf("%v+", ul.Items)
```

Comment:
Do we want to keep this? If so, `t.Logf()` may be better suited so it only shows when you provide the verbose flag in `go test`

Reply:
Good catch - I'll remove those. We don't want 'em! 😄